pcsx2/PluginManager.cpp

```diff
@@ -171,6 +171,7 @@ _GSinitReadFIFO    GSinitReadFIFO;
 _GSinitReadFIFO2   GSinitReadFIFO2;
 _GSchangeSaveState GSchangeSaveState;
 _GSgetTitleInfo2   GSgetTitleInfo2;
+_GSgetImageSize    GSgetImageSize;
 _GSmakeSnapshot	   GSmakeSnapshot;
```

Comment:
Just a small ocd nitpick I saw in the file. Some of them have weird spacing. Do you wanna update them as well since they kinda look out of place since ?

You don't have to do it if you don't want to :stuck_out_tongue: